cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -4,289 +4,750 @@
             [cljfmt.core :refer [reformat-string default-line-separator
                                  normalize-newlines find-line-separator
                                  replace-newlines wrap-normalize-newlines]]
-            [clojure.string :as str]))
+            [cljfmt.test-util.common :as common])
+  #?(:clj (:require [cljfmt.test-util.clojure])
```

Comment:
Rather than having two `:require` statements, it would be better just to have one.